git.lua

```diff
@@ -112,6 +112,30 @@ local function branches(token)
     return res
 end
 
+local function alias(token)
+    local res = {}
+
+    -- Try to resolve .git directory location
+    local git_dir = get_git_dir()
+
+    if git_dir == nil then return res end
+
+    f = assert (io.popen ("git config --get-regexp alias"))
+
+    for line in f:lines() do
+        local s, e = line:find(" ", 1, true)
```

Comment:
Please consider rework this using `string.match` and regexp. I'm not a fan of RegExps but in this case it'll be more readable. This can look like
```lua
local alias, command = line:match("alias%.(.+)%s+(.+)$")
```